lib/libzfs_core/libzfs_core.c

```diff
@@ -340,6 +344,12 @@ lzc_exists(const char *dataset)
 	return (ioctl(g_fd, ZFS_IOC_OBJSET_STATS, &zc) == 0);
 }
 
+int
+lzc_sync(const char *pool_name)
```

Comment:
maybe even `nvlist_t **opts`? Not sure why we need to allocate a nvlist though, we can just ignore the argument provided to lzc_sync().